.github/PULL_REQUEST_TEMPLATE/strawman.md

```diff
@@ -0,0 +1,19 @@
+---
+name: 💡 Proposal (RFC 1)
```

Comment:
did you check how those templates and prs would show up? on some test repo? Last time I checked GH supported only one PR template

Reply:
No, I haven't. And thanks for bringing the comment up. This file is wrong. It says "strawman" but in fact, it's a proposal RFC 1.

Reply:
You can choose the template to use by adding the `?template=issue_template.md`. Unfortunately, there is no UI for that yet.

More info at https://docs.github.com/en/github/managing-your-work-on-github/about-automation-for-issues-and-pull-requests-with-query-parameters#supported-query-parameters

Reply:
Replaced Proposal (RFC 1) with Strawman (RFC 0).

Reply:
@smoya how would it work with PR, no example in docs? I think we need one template with all sections and contributor just need to leave the one that is relevant

Reply:
If it's gonna be always the same template, yes one should be enough. Otherwise you can also set a default one but also have more templates people can use (by adding the query parameter)

Reply:
the thing is that if for now, the only way to use different PR templates is to use some mysterious query param, it is not very helpful. I mean who would even remember the names of templates anyway 🤷🏼 this is why better to consolidate all into one template that will always show up

Reply:
Yup, makes sense as the ux doesnt help this time 👍

Reply:
Ok, I'll remove them. It doesn't make sense with the PR workflow if there's no UI for that.

Reply:
How about now?